src/configs/controllers/configController.ts

```diff
@@ -17,6 +23,42 @@ function configMapper(config: Config): components['schemas']['config'] {
   };
 }
 
+const sortFieldsMap = new Map<string, keyof Omit<Config, 'config'>>(
+  Object.entries({
+    /* eslint-disable @typescript-eslint/naming-convention */
+    'config-name': 'configName',
+    version: 'version',
+    'created-at': 'createdAt',
+    'schema-id': 'schemaId',
+    'created-by': 'createdBy',
+    /* eslint-enable @typescript-eslint/naming-convention */
+  })
+);
+
+function sortOptionParser(sortArray: components['parameters']['SortQuery']): { field: keyof Omit<Config, 'config'>; order: 'asc' | 'desc' }[] {
```

Comment:
Consider using an interface for the sort option instead of writing this every time
```suggestion
interface SortOption {
  field: SortableFields;
  order: 'asc' | 'desc';
}
function sortOptionParser(sortArray: components['parameters']['SortQuery']): SortOption[] {
```

Reply:
changed